### PR TITLE
Document GDScript `@export_tool_button`

### PR DESCRIPTION
**Pull request by 52SW**
Add @export_tool_button for GDscript to the documention for https://github.com/godotengine/godot/pull/96290 .

### CODE REVIEW

**Review by tetrapod00 — CHANGES_REQUESTED** [review]
This looks great once changes are applied.

I can think of some possible improvements to make in future PRs:
- Link to tool script documentation
- Mention that the scene may need to be reloaded for changes to propagate
- Further document what the label and icon parameters do,
- Include a second example with advanced usage.

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 1]
```diff
@@ -480,6 +480,24 @@ for a list of parameters and their allowed values.
     When using ``@export_custom``, GDScript does not perform any validation on
     the syntax. Invalid syntax may have unexpected behavior in the inspector.
 
+``@export_tool_button``
+-----------------------
+
+If you need to create a clickable inspector button, you can use ``@export_tool_button``. 
+This would export a Callable property as a clickable button. When the button is pressed, the callable is called. 
+
+Export a button with label ``"Hello"`` and icon ``"Callable"``. When you press it, it will print ``"Hello world!"``.
```
[the remark this answers is not in the document]
Comment (52SW) [reply tutorials/scripting/gdscript/gdscript_exports.rst:489]: Thanks for your advices, I would make some changes and update my pr soon. Have a nice day!

**Review by tetrapod00 — APPROVED** [review]
Looks good to me. 

Another reviewer still needs to take a look, and they might ask you to make additional changes.

File: tutorials/scripting/gdscript/gdscript_exports.rst [hunk 2]
```diff
@@ -480,6 +480,24 @@ for a list of parameters and their allowed values.
     When using ``@export_custom``, GDScript does not perform any validation on
     the syntax. Invalid syntax may have unexpected behavior in the inspector.
 
+``@export_tool_button``
+-----------------------
+
+If you need to create a clickable inspector button, you can use ``@export_tool_button``.
+This exports a Callable property as a clickable button. When the button is pressed, the callable is called.
```
[the remark this answers is not in the document]
Comment (52SW) [reply tutorials/scripting/gdscript/gdscript_exports.rst]: Thanks! I've changed these.

**Review by dalexeev — APPROVED** [review]

**Review by skyace65 — APPROVED** [review]

**Comment by skyace65** [comment]
Thanks! And congrats on your first merged PR! In the future please try and keep PRs to 1 commit, I did a squash and merge this time so it's fine but keep that in mind for the future. If you don't know how to do that we have a guide [here](https://docs.godotengine.org/en/stable/contributing/workflow/pr_workflow.html#modifying-a-pull-request).

**Pull request merged**